src/chain/conviction-voting.ts

```diff
@@ -109,7 +109,7 @@ function toVotings(
 }
 
 export async function getVotingFor(
-  api: ApiPromise,
+  api: { query: QueryableStorage<'promise'> },
```

Comment:
Any reason the type is not set to ApiPromise?

Reply:
This is so that it works with `api.at`